src/comment/comment.service.ts

```diff
@@ -0,0 +1,51 @@
+import { Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { Comment } from './entities/comment.entity';
+import { CreateCommentDto } from './dto/create-comment-dto;
+
+@Injectable()
+export class CommentService {
+  constructor(
+    @InjectRepository(Comment)
+    private commentRepository: Repository<Comment>,
+  ) {}
+
+  async create(createCommentDto: CreateCommentDto): Promise<Comment> {
+    const comment = this.commentRepository.create(createCommentDto);
+    return this.commentRepository.save(comment);
+  }
+
+  async findAll(): Promise<Comment[]> {
+    return this.commentRepository.find();
+  }
+
+  async findById(id: number): Promise<Comment> {
+    return this.commentRepository.findOne(id);
+  }
+
+  async update(id: number, updateCommentDto: CreateCommentDto): Promise<Comment> {
+    const comment = await this.commentRepository.preload({ id: +id, ...updateCommentDto });
+    if (!comment) {
+      throw new Error('Comment not found');
```

Comment:
you should throw [standard exceptions](https://docs.nestjs.com/exception-filters) which are provided by nestjs. if you try to throw an error in here, nestjs won't be catch this error and thus will shutdown RMC's backend.